### ISSUE TITLE
archive.org arxiv-bulk down?

### ISSUE THREAD

**Issue by SamaHadhoud** [potamides/arxiv-latex-extract#3]
The archive.org arxiv-bulk seems to be down is there any solution for that?
Thanks in advance.

**Comment by potamides** [comment]
The temporary alternative would be to rely on Amazon S3 for all buckets. The downside is of course that this is not free of charge.

**Comment by SamaHadhoud** [comment]
yes, I get it. do you have any idea if the archive will be up again soon or not?

**Comment by potamides** [comment]
Seems like the archive is up again!

**Comment by SamaHadhoud** [comment]
I think they have a major problem or smth  as it is down again, so I went with the Amazon S3 option. Thanks!
